docs/resources/v2-conformance-statement.md

```diff
@@ -0,0 +1,818 @@
+| Exact Match | All supported attributes | {attributeID}={value1} |
+| Fuzzy Match | PatientName, ReferringPhysicianName | Matches any component of the name which starts with the value. |
+
+#### Attribute ID
+
+Tags can be encoded in a number of ways for the query parameter. We have partially implemented the standard as defined in [PS3.18 6.7.1.1.1](http://dicom.nema.org/medical/dicom/2019a/output/chtml/part18/sect_6.7.html#sect_6.7.1.1.1). The following encodings for a tag are supported:
+
+| Value            | Example          |
+| :--------------- | :--------------- |
+| {group}{element} | 0020000D         |
+| {dicomKeyword}   | StudyInstanceUID |
+
+Example query searching for instances:
+`../instances?Modality=CT&00280011=512&includefield=00280010&limit=5&offset=0`
+
+### Search Response
+
+The response will be an array of DICOM datasets. Depending on the resource, by *default* the following attributes are returned:
+
+#### Default Study tags
+
+| Tag          | Attribute Name |
+| :----------- | :------------- |
+| (0008, 0020) | StudyDate |
+| (0008, 0050) | AccessionNumber |
+| (0008, 1030) | StudyDescription |
+| (0009, 0090) | ReferringPhysicianName |
+| (0010, 0010) | PatientName |
+| (0010, 0020) | PatientID |
+| (0010, 0030) | PatientBirthDate |
+| (0020, 000D) | StudyInstanceUID |
+
+#### Default Series tags
+
+| Tag          | Attribute Name |
+| :----------- | :------------- |
+| (0008, 0060) | Modality |
+| (0008, 1090) | ManufacturerModelName |
+| (0020, 000E) | SeriesInstanceUID |
+| (0040, 0244) | PerformedProcedureStepStartDate |
+
+#### Default Instance tags
+
+| Tag          | Attribute Name |
+| :----------- | :------------- |
+| (0008, 0018) | SOPInstanceUID |
+
+If `includefield=all`, below attributes are included along with default attributes. Along with default attributes, this is the full list of attributes supported at each resource level.
+
+#### Additional Study tags
+
+| Tag          | Attribute Name |
+| :----------- | :------------- |
+| (0008, 0005) | SpecificCharacterSet |
+| (0008, 0030) | StudyTime |
+| (0008, 0056) | InstanceAvailability |
+| (0008, 0201) | TimezoneOffsetFromUTC |
+| (0008, 0063) | AnatomicRegionsInStudyCodeSequence |
+| (0008, 1032) | ProcedureCodeSequence |
+| (0008, 1060) | NameOfPhysiciansReadingStudy |
+| (0008, 1080) | AdmittingDiagnosesDescription |
+| (0008, 1110) | ReferencedStudySequence |
+| (0010, 1010) | PatientAge |
+| (0010, 1020) | PatientSize |
+| (0010, 1030) | PatientWeight |
+| (0010, 2180) | Occupation |
+| (0010, 21B0) | AdditionalPatientHistory |
+| (0010, 0040) | PatientSex |
+| (0020, 0010) | StudyID |
+
+#### Additional Series tags
+
+| Tag          | Attribute Name |
+| :----------- | :------------- |
+| (0008, 0005) | SpecificCharacterSet |
+| (0008, 0201) | TimezoneOffsetFromUTC |
+| (0020, 0011) | SeriesNumber |
+| (0020, 0060) | Laterality |
+| (0008, 0021) | SeriesDate |
+| (0008, 0031) | SeriesTime |
+| (0008, 103E) | SeriesDescription |
```

Comment:
How do we decide what is in the default vs. additional tags?  For example, we include `StudyDescription` in the study by default, but `SeriesDescription` is not included with the series.

Reply:
Things we index by defaults stayed in as defaults